ui/src/app/components/workflows/workflow/dynamic-parts/guid-part/guid-part.component.html

```diff
@@ -22,7 +22,7 @@
          [readOnly]="true"
          [ngModel]="value"
          (ngModelChange)="modelChanged($event)"
-         [size]="partValidationSafe.maxLength"
+         [size]="partValidationSafe.maxLength < 100 ? partValidationSafe.maxLength : 100"
```

Comment:
Please extract the value 100 to a variable
Same in `key-string-value-part`, `string-part` and `string-sequence-part`

Reply:
Fixedd